src/lv_message.cc

```diff
@@ -356,10 +356,12 @@ namespace grpc_labview
             {
                 v = std::static_pointer_cast<LVRepeatedMessageValue<std::string>>((*it).second);
             }
-            protobuf_ptr -= 1;
+
+            auto tagSize = CalculateTagWireSize(tag);
```

Comment:
NOTE: This no longer assumes the field can be encoded in a one byte tag. This change was made for both repeated string and message fields for both the original serialization code path and the efficient message code path.